bot/seasons/valentines/be_my_valentine.py

```diff
@@ -0,0 +1,232 @@
+            # This command is only DM specific
+            msg = "You are not supposed to use this command in the server, DM the command to the bot."
+            return await ctx.send(msg)
+
+        if user == ctx.author:
+            # Well a user cant valentine himself/herself.
+            await ctx.send('Come on dude, you cant send a valentine to yourself :expressionless:')
+
+        if user is None:
+            # just making sure that the random dosent pick up the same user(ctx.author)
+            user = self.random_user(ctx.author)
+
+        if valentine_type is None:
+            valentine, title = self.random_valentine()
+
+        elif valentine_type.lower() in ['p', 'poem']:
+            valentine = self.valentine_poem()
+            title = f'A poem dedicated to'
+
+        elif valentine_type.lower() in ['c', 'compliment']:
+            valentine = self.valentine_compliment()
+            title = f'A compliment for'
+
+        else:
+            # in this case, the user decides to type his own valentine.
+            valentine = valentine_type
+            title = f'A message for'
+
+        embed = discord.Embed(
+            title=f'{emoji_1}{title} {user.display_name}{emoji_2}',
+            description=f'{valentine} \n **{emoji_2}From anonymous{emoji_1}**',
+            color=Colours.pink
+        )
+        await ctx.author.send(f"Your message has been sent to {user}")
+        await user.send(embed=embed)
+
+    @staticmethod
+    def random_user(author):
+        USER_LOVEFEST.remove(author)
+        user = random.choice(USER_LOVEFEST)
+        USER_LOVEFEST.append(author)
+        return user
+
+    @staticmethod
+    def random_emoji():
+        EMOJI_1 = random.choice(HEART_EMOJIS)
+        EMOJI_2 = random.choice(HEART_EMOJIS)
+        return EMOJI_1, EMOJI_2
+
+    def random_valentine(self):
+        """
+        Grabs a random poem or a compliment (any message).
+        """
+        valentine_poem = random.choice(self.valentines['valentine_poems'])
+        valentine_compliment = random.choice(self.valentines['valentine_compliments'])
+        random_valentine = random.choice([valentine_compliment, valentine_poem])
+        if random_valentine == valentine_poem:
+            title = 'A poem dedicated to'
+        else:
+            title = 'A compliment for '
+        return random_valentine['message'], title
+
+    def valentine_poem(self):
+        """
+        Grabs a random poem.
+        """
+        valentine_poem = random.choice(self.valentines['valentine_poems'])
+        return valentine_poem['message']
+
+    def valentine_compliment(self):
+        """
+        Grabs a random compliment.
+        """
+        valentine_compliment = random.choice(self.valentines['valentine_compliments'])
+        return valentine_compliment['message']
+
+
+def setup(bot):
+    JSON_FILE = open(Path('bot', 'resources', 'valentines', 'bemyvalentine_valentines.json'), 'r', encoding="utf8")
```

Comment:
The JSON load should be a method of your `BeMyValentine` and called as part of its `__init__`. 

You'll want to use a [context manager](https://docs.python.org/3/reference/compound_stmts.html#with) (`with open(...) as f:`) as well. This protects against errors during load and closes the file automatically when you're done with the file or if there is an error. You can see an example in the [Python 3 Tutorial](https://docs.python.org/3/tutorial/inputoutput.html#reading-and-writing-files).

`Path` also provides an `Path.open()` method. You can see an example of this in [`pathlib`'s documentation](https://docs.python.org/3/library/pathlib.html#pathlib.Path.open)